src/platform_impl/ios/app_state.rs

```diff
@@ -662,6 +662,28 @@ fn handle_user_events(mtm: MainThreadMarker) {
     }
 }
 
+pub(crate) fn send_occluded_event_for_all_windows(application: &UIApplication, occluded: bool) {
+    let mtm = MainThreadMarker::from(application);
+
+    let mut events = Vec::new();
+    #[allow(deprecated)]
+    for window in application.windows().iter() {
+        if window.is_kind_of::<WinitUIWindow>() {
```

Comment:
move this to `filter`

Reply:
I'd rather not at this time, it's a safety condition for the `unsafe` just below, and moving it to `filter` makes it seem like they're less connected.